Source/carthage/Printer.swift

```diff
@@ -0,0 +1,40 @@
+import Foundation
+
+public protocol Printer {
+	func println()
+	func println(object: Any)
+	func print(object: Any)
+}
+
+internal struct ThreadSafePrinter: Printer {
+	private static let outputQueue = { () -> dispatch_queue_t in
+		let queue = dispatch_queue_create("org.carthage.carthage.outputQueue", DISPATCH_QUEUE_SERIAL)
+		dispatch_set_target_queue(queue, dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_HIGH, 0))
+
+		atexit_b {
```

Comment:
You'll note that I pretty much copied this verbatim from Extensions.swift (see https://github.com/Carthage/Carthage/blob/master/Source/carthage/Extensions.swift#L19-L49)